mscore/menus.cpp

```diff
@@ -1308,6 +1308,65 @@ void MuseScore::setBasicPalette()
       }
 
 //---------------------------------------------------------
+//   genCreateMenu
+//---------------------------------------------------------
+
+QMenu* MuseScore::genCreateMenu(QWidget* parent)
```

Comment:
Okay, I'll take care of it.

On Tue, Aug 23, 2016 at 11:17 PM, IsaacWeiss <notifications@github.com>
wrote:

> In mscore/menus.cpp
> <https://github.com/musescore/MuseScore/pull/2789#discussion_r75916578>:
>
> > @@ -1308,6 +1308,65 @@ void MuseScore::setBasicPalette()
> >        }
> >
> >  //---------------------------------------------------------
> > +//   genCreateMenu
> > +//---------------------------------------------------------
> > +
> > +QMenu* MuseScore::genCreateMenu(QWidget* parent)
>
> That is correct. I moved the Add menu into mscore/musescore.cpp where the
> others were. So this whole thing should be removed.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/musescore/MuseScore/pull/2789/files/b875e9fe5be4d970b59c6721f2e3a62469df0af3#r75916578>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGf7yrLLgmHTYPFm-0lkvRMoNsq5uH8bks5qizItgaJpZM4JpFpz>
> .
>



-- 
Ruchit Agrawal.
IIIT Hyderabad.